app/views/application/_email-confirmation-reminder.html.erb

```diff
@@ -0,0 +1,6 @@
+<section class="email-reminder" aria-label="Notice" role="region">
```

Comment:
Interesting, what does an `aria-label="Notice"` do? Is it unusual that it is a capitalised?

Reply:
I copied this from our notice component (https://components.publishing.service.gov.uk/component-guide/notice). Aria labels are used to explain what an element is there for, which in this case is a notice. It helps users find and understand what this is for.

I was thinking of doing this text as a heading, but there's no text for it to be the heading of, and it would potentially conflict with the rest of the headings on the page, so I went with our existing practice.

Reply:
Ace thanks for explaining!

Reply:
GOV.UK Frontend's latest release includes a new notification component (https://design-system.service.gov.uk/components/notification-banner/) so I might see if we can use that instead.